gemrb/core/System/String.h

```diff
@@ -23,6 +23,9 @@
 
 #include <string>
 
+#define FMT_HEADER_ONLY
+#include <fmt/format.h>
```

Comment:
You forgot to add these file(s).

Reply:
I didn't forget, I'm unsure of how we wanted to integrate it. The license is indeed permissive enough to just dump it in our code or do a git submodule. How/where do we want it?

Reply:
I suggest we just keep it simple and dump the specific file (or folder) we need into gemrb/includes. It's unlikely we'll have to update it often, so a submodule seems overkill.